CODE_OF_CONDUCT.md

```diff
@@ -95,71 +108,93 @@ We will not accept harassment or other exclusionary behaviours, such as:
 - Other unethical or unprofessional conduct
 - Advocating for, or encouraging, any of the above behaviors
 
-### Reporting Guidelines
+The conda Organization prioritizes marginalized people’s safety over privileged people’s comfort. The conda CoC Committee reserves the right not to act on complaints including, but not limited to:
+
+* ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’.
+* Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
+* Communicating in a ‘tone’ you don’t find congenial.
+* Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions.
 
-If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The conda Organization leadership team takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+## Behavior Outside of conda Organization Spaces
 
-**All reports will be kept confidential.**
+The CoC Committee does not influence behavior and membership in spaces outside the conda Organization. However, if you are being harassed by a member of the conda community outside our spaces, you may still report it to the CoC Committee. We will take all good-faith reports of harassment by conda community members seriously. This includes harassment outside our spaces and harassment that took place at any point in time.
 
-In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
+The CoC Committee reserves the right to exclude people from conda Organization spaces based on their past behavior, including behavior outside conda Organization spaces and behavior towards people who are not in the conda community.  
 
-All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances.  The conda Organization team commits to maintaining confidentiality with regard to the reporter of an incident.
+# Confidentiality and Public Statements to the Community
 
-For possibly unintentional breaches of the code of conduct, you may want to respond to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please report the issue to the conda Organization directly, or ask [PROJECT CoC Point Person(s)] for advice in confidence ([PROJECT CoC Point Person(s) Email address or other means of contact]). Complete contact information is below, under "How to Submit a Report."
+The CoC Committee will keep the identity of the reporter confidential. 
```

Comment:
How does this apply in cases of mandatory reporters serving on the CoC Committee? This seems like a statement that may not be able to be kept.

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
@CJ-Wright, I hadn't thought about mandatory reporters.

**Background:** Some universities (and maybe some companies) have policies that if you are aware of certain bad behaviors by a fellow employee then you must report the behavior to the university (/company). That report might be expected to include the identity of the reporter. If the person being reported works at the same place as a committee member, then that committee member is now in a very hard place.

Does anyone know how common it is for these policies to not allow statements like "This was reported to me in my capacity as an `Organization Name` Code of Conduct committee member, **by a person who wishes to remain anonymous**"?

Reply:
> If the person is uncomfortable, they can seek out another member.

the tricky part of mandatory reporting is that once you, the mandatory reporter, are made aware of a situation you have to report it (the mandatory part). so the person who reports it doesn't really have the opportunity to keep it confidential once you tell a mandatory reporter. at least that's my understanding

Reply:
Fair.